SQL/New_patches/2018-09-07_EEG_tables.sql

```diff
@@ -0,0 +1,259 @@
+-- Create a physiological_modality table
+CREATE TABLE `physiological_modality` (
+  `PhysiologicalModalityID` TINYINT UNSIGNED NOT NULL AUTO_INCREMENT,
+  `PhysiologicalModality`   VARCHAR(50)      NOT NULL,
+  PRIMARY KEY (`PhysiologicalModalityID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+-- Create a physiological_output_type table
+CREATE TABLE `physiological_output_type` (
+  `PhysiologicalOutputTypeID` INT(5) UNSIGNED NOT NULL AUTO_INCREMENT,
+  `OutputType`                VARCHAR(20)     NOT NULL,
+  `OutputTypeDescription`     VARCHAR(255),
+  PRIMARY KEY (`PhysiologicalOutputTypeID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+-- Create a physiological_file table
+CREATE TABLE `physiological_file` (
+  `PhysiologicalFileID`       INT(10)    UNSIGNED  NOT NULL      AUTO_INCREMENT,
+  `PhysiologicalModalityID`   TINYINT(3) UNSIGNED  DEFAULT NULL,
+  `Caveat`                    TINYINT(1) UNSIGNED  DEFAULT 0,
+  `PhysiologicalOutputTypeID` INT(5)     UNSIGNED  NOT NULL,
+  `SessionID`                 INT(10)    UNSIGNED  NOT NULL,
+  `InsertTime`                INT(10)    UNSIGNED  NOT NULL,
+  `FileType`                  VARCHAR(12)          DEFAULT NULL,
+  `AcquisitionTime`           VARCHAR(20)          DEFAULT NULL,
+  `InsertedByUser`            VARCHAR(50)          NOT NULL,
+  `File`                      VARCHAR(255)         NOT NULL,
+  PRIMARY KEY (`PhysiologicalFileID`),
+  CONSTRAINT `FK_session_ID`
+    FOREIGN KEY (`SessionID`)
+    REFERENCES `session` (`ID`),
+  CONSTRAINT `FK_ImagingFileTypes_type`
+    FOREIGN KEY (`FileType`)
+    REFERENCES `ImagingFileTypes` (`type`),
+  CONSTRAINT `FK_physiological_modality_PhysiologicalModalityID`
+    FOREIGN KEY (`PhysiologicalModalityID`)
+    REFERENCES `physiological_modality` (`PhysiologicalModalityID`),
+  CONSTRAINT `FK_physiological_output_type_PhysiologicalOutputTypeID`
+    FOREIGN KEY (`PhysiologicalOutputTypeID`)
+    REFERENCES `physiological_output_type` (`PhysiologicalOutputTypeID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+
+-- Create a physiological_parameter_file table that will store all JSON
+-- information that accompanies the BIDS physiological dataset
+CREATE TABLE `physiological_parameter_file` (
+  `PhysiologicalParameterFileID` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+  `PhysiologicalFileID`          INT(10) UNSIGNED NOT NULL,
+  `ParameterTypeID`              INT(10) UNSIGNED NOT NULL,
+  `InsertTime`                   INT(10) UNSIGNED NOT NULL,
+  `Value`                        VARCHAR(255),
```

Comment:
Could you be more precise for "value"?

Reply:
What type of value.

Reply:
It could be anything (blake2b of the registered file, information stored in the side car JSON file etc...). Basically, this is the `Value` associated to the `ParameterTypeID` present in the `parameter_type` table where the description and name of the parameter is stored.

Reply:
Ok then, a generic attribute name is justified.